doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -718,6 +718,37 @@ the toplevel, and using them in source files is discouraged.
    the :ref:`command line option <command-line-options>` :n:`-I @string` or by the command :cmd:`Add
    ML Path` `@string` (cf. :cmd:`Declare ML Module`).
 
+.. _extra_dependencies:
+
+Extra Dependencies
+------------------
+
+Dependencies on external files, i.e. non ``.v`` files, can be declared as
+follows:
+
+.. cmd:: From @dirpath Extra Dependency @string {? as @ident }
+   :name: From … Dependency
+
+   Coq checks that a file name :n:`@string` exists in one, and only one,
+   directory associated with :n:`@dirpath`. It does not recursively
+   scan subdirectories but :n:`@string` can contain the directory separator
+   ``/`` and hence denote a file in a subdirectory.
+   The ``coqdep`` tool generates a dependency on the external file for the
+   current ``.v`` file.
+   Note that path elements in :n:`@string` must be valid Coq identifiers,
+   e.g. they cannot contain characters such as ``-`` or ``,``.
```

Comment:
```suggestion
   Adds an additional dependency of the current `.v`  file on an external file.  This
   information is included in the ``coqdep`` tool generated list of dependencies.
   The file name :n:`@string` must exist relative to only one of the top directories
   associated with :n:`@dirpath`.  :n:`@string` can include directory separators
   (``/``) to select a file in a subdirectory.   
   Path elements in :n:`@string` must be valid Coq identifiers, e.g. they cannot
   contain characters such as ``-`` or ``,``.  See :ref:`lexical-conventions`.
```

Hmm, I commented on this before but maybe I forgot to click "Add review comment"

Is it an error if `@string` refers to a .v file?  Does reporting it as an error really matter?

If `@string` starts with `..`, is that valid?  Worth complaining about?  How about
a string starting with `/`?

Identifier syntax is defined [here](https://coq.inria.fr/distrib/current/refman/language/core/basic.html#lexical-conventions).  Unicode characters are OK in `@string`?